scss/widgets/base/_colorView.scss

```diff
@@ -0,0 +1,17 @@
+
+@mixin dx-base-colorview-styles($handle-color, $handle-border-color) {
+    .dx-colorview-palette-handle {
```

Comment:
I think we should use 2 spaces like in TS.

Reply:
https://github.com/DevExpress/DevExtreme/pull/13518/commits/d1edef37cf34f88f81767446af07f74e38914135